### ISSUE TITLE
Show conflicting Axioms in detail

### ISSUE THREAD

**Issue by SeeBasTStick** [hhu-stups/prob-issues#312]
When working with Event-B contexts it would be useful why and which Axioms are in conflict:

Example:
``
de.prob.exception.ProBError: ProB reported Errors
ProB returned error messages:
Error: AXIOMS are unsatisfiable (but some CONSTANTS valued)
Message: AXIOMS are *not* true and no value was found (deterministically) for the following CONSTANT: abstract_triples
``

Here we only see the constant which could not be instantiated but there is no hint which axiom could not be satisfied.

Edit: I notice that you can unsat core, so maybe this option should be moved into the error message field as an alternative (PRoB2Ui).

**Comment by leuschel** [comment]
Yes, the unsat core is the solution: its purpose is exactly to find a minimal core of conflicting axioms.

Sometimes, when you are lucky, ProB is able to infer a conflicting root cause directly without running the unsat core algorithm. In this case that is reported to the user in the message.
